drivers/lsm6ds3/include/lsm6ds3-internal.h

```diff
@@ -0,0 +1,210 @@
+#define LSM6DS3_CTRL1_XL_BW_200HZ         (0x01)
+#define LSM6DS3_CTRL1_XL_BW_100HZ         (0x02)
+#define LSM6DS3_CTRL1_XL_BW_50HZ          (0x03)
+/** @} */
+
+/**
+ * @name Masks for the LSM6DS3 CTRL2_G registers
+ * @{
+ */
+#define LSM6DS3_CTRL2_G_ODR_POWEROFF      (0x00)
+#define LSM6DS3_CTRL2_G_ODR_13HZ          (0x01)
+#define LSM6DS3_CTRL2_G_ODR_26HZ          (0x02)
+#define LSM6DS3_CTRL2_G_ODR_52HZ          (0x03)
+#define LSM6DS3_CTRL2_G_ODR_104HZ         (0x04)
+#define LSM6DS3_CTRL2_G_ODR_208HZ         (0x05)
+#define LSM6DS3_CTRL2_G_ODR_416HZ         (0x06)
+#define LSM6DS3_CTRL2_G_ODR_833HZ         (0x07)
+#define LSM6DS3_CTRL2_G_ODR_1660HZ        (0x08)
+#define LSM6DS3_CTRL2_G_FS_245DPS         (0x00)
+#define LSM6DS3_CTRL2_G_FS_500DPS         (0x01)
+#define LSM6DS3_CTRL2_G_FS_1000DPS        (0x02)
+#define LSM6DS3_CTRL2_G_FS_2000DPS        (0x03)
+#define LSM6DS3_CTRL2_G_FS_125_DISABLED   (0x00)
+#define LSM6DS3_CTRL2_G_FS_125_ENABELD    (0x01)
+/** @} */
+
+/**
+ * @name Masks for the LSM6DS3 CTRL3_C register
+ * @{
+ */
+#define LSM6DS3_CTRL3_C_BOOT              (0x80)
+#define LSM6DS3_CTRL3_C_BDU               (0x40)  /* block data update */
+/** @} */
+
+/**
+ * @name Masks for the LSM6DS3 CTRL4_C register
+ * @{
+ */
+#define LSM6DS3_CTRL4_C_SLEEP_G           (0x40)
+#define LSM6DS3_CTRL4_C_TEMP_EN           (0x10)
+/** @} */
+
+/**
+ * @name Masks for the LSM6DS3 CTRL9_XL register
+ * @{
+ */
+#define LSM6DS3_CTRL9_XL_Z_EN             (0x20)
+#define LSM6DS3_CTRL9_XL_Y_EN             (0x10)
+#define LSM6DS3_CTRL9_XL_X_EN             (0x08)
+#define LSM6DS3_CTRL9_XL_PEDO_RST_STEP    (0x02)
+/** @}*/
+
+/**
+ * @name Masks for the LSM6DS3 CTRL9_XL register
+ * @{
+ */
+#define LSM6DS3_CTRL10_G_Z_EN             (0x20)
+#define LSM6DS3_CTRL10_G_Y_EN             (0x10)
+#define LSM6DS3_CTRL10_G_X_EN             (0x08)
+#define LSM6DS3_CTRL10_G_PEDO_RST_STEP    (0x02)
+/** @}*/
+
+/**
+ * @name LSM6DS3 ouput register of the accelerometer
+ * @{
+ */
+#define LSM6DS3_REG_OUT_X_L_XL                 (0x28)
+#define LSM6DS3_REG_OUT_X_H_XL                 (0x29)
+#define LSM6DS3_REG_OUT_Y_L_XL                 (0x2A)
+#define LSM6DS3_REG_OUT_Y_H_XL                 (0x2B)
+#define LSM6DS3_REG_OUT_Z_L_XL                 (0x2C)
+#define LSM6DS3_REG_OUT_Z_H_XL                 (0x2D)
+/** @} */
+
+/**
+ * @name LSM6DS3 output register of the gyroscope
+ * @{
+ */
+#define LSM6DS3_REG_OUT_X_L_G                  (0x22)
+#define LSM6DS3_REG_OUT_X_H_G                  (0x23)
+#define LSM6DS3_REG_OUT_Y_L_G                  (0x24)
+#define LSM6DS3_REG_OUT_Y_H_G                  (0x25)
+#define LSM6DS3_REG_OUT_Z_L_G                  (0x26)
+#define LSM6DS3_REG_OUT_Z_H_G                  (0x27)
+/** @} */
+
+/**
+ * @name LSM6DS3 temperature registers
+ * @{
+ */
+#define LSM6DS3_OUT_TEMP_L                (0x20)
+#define LSM6DS3_OUT_TEMP_H                (0x21)
+/*
+ * TODO: temperature enable macro
```

Comment:
Well, if there is no technical TODO left, then the last TODO is to remove the word "TODO" out of the comment.